app/connectors/EnrolmentsConnector.scala

```diff
@@ -17,21 +17,34 @@
 package connectors
 
 import config.EnrolmentsConfig
-import connectors.EnrolmentsHttpParser.{EnrolmentResultsResponse, EnrolmentsResponseReads}
 import logging.Logging
+import models.enrolments.SubscriberRequest
 import play.api.http.HeaderNames
-import uk.gov.hmrc.domain.Vrn
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpErrorFunctions}
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpErrorFunctions, HttpResponse}
 
+import java.util.UUID
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
 
 class EnrolmentsConnector @Inject()(enrolments: EnrolmentsConfig, httpClient: HttpClient)
                                    (implicit ec: ExecutionContext) extends HttpErrorFunctions with Logging {
 
+  private implicit val emptyHc: HeaderCarrier = HeaderCarrier()
   val headers = Seq(
     HeaderNames.AUTHORIZATION -> s"Bearer ${enrolments.authorizationToken}"
   )
 
+  private val etmpId = UUID.randomUUID().toString
```

Comment:
(At a glance comment - delete if not appropriate)

It looks like this is defined outside of the scope of the method where it is used. If you want this UUID to be different every time `confirmEnrolment` is called then it should be instantiated inside the method 👍🏻